acceptance/bundle/variables/resolve-nonstrings/script

```diff
@@ -0,0 +1,3 @@
+export BUNDLE_VAR_no_alert_for_skipped_runs=false
+export BUNDLE_VAR_max_workers=2
+$CLI bundle validate -o json --var no_alert_for_canceled_runs=true --var min_workers=1 --var spot_bid_max_price=0.5 | jq '{ variables, jobs: .resources.jobs }'
```

Comment:
Is the mix of env var + flag random? Could be more clear by adding 2 calls, one with env vars and one with flags.

Reply:
It's a mix to cover different code paths.

Let me add one more, actually, where --var shadows env var https://github.com/databricks/cli/pull/2143/commits/01ebffdac39c41e376fed310cc50e5cf053dbd50